core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -1925,6 +1927,30 @@ public void testDropCatalog()
                 new DropCatalog(location(1, 1), new Identifier(location(1, 14), "some catalog that contains space", true), false, false));
     }
 
+    @Test
+    public void testRenameCatalog()
+    {
+        assertStatement("ALTER CATALOG a RENAME TO b", new RenameCatalog(new Identifier("a"), new Identifier("b")));
```

Comment:
`assertStatement` is deprecated. Use `assertThat(statement(...))`. See other tests for inspiration.